### PR TITLE
nilrt.conf: Bump version number and feed name

### PR DESCRIPTION
**Pull request by billpittman**
Bump DISTRO_VERSION to 8.11.
Bump NILRT_FEED_NAME to 2021.8.

Signed-off-by: Bill Pittman <bill.pittman@ni.com>

### CODE REVIEW

File: conf/distro/nilrt.conf [hunk 1]
```diff
@@ -2,7 +2,7 @@ require nilrt.inc
 
 DISTRO_NAME = "NI Linux Real-Time"
 
-DISTRO_VERSION = "8.10"
+DISTRO_VERSION = "8.11"
 
 NILRT_FEED_NAME = "2021.5"
```
Comment (amstewart) [inline conf/distro/nilrt.conf]: Could you also change the `FEED_NAME` to `2021.8`, while you're at it.

**Review by amstewart — CHANGES_REQUESTED** [review]

**Review by chaitu236 — APPROVED** [review]
ShipIt assuming FEED_NAME is changed

File: conf/distro/nilrt.conf (hunk 1 above)
Comment (billpittman) [reply conf/distro/nilrt.conf]: Sure, but that change shouldn't be cherry picked into nilrt/21.5/sumo right?  This change should be.  So assuming I'm right, lets make the NILRT_FEED_NAME change as another PR that is NOT cherry-picked into nilrt/21.5/sumo.  Sound good?

Comment (amstewart) [reply conf/distro/nilrt.conf]: The `DISTRO_VERSION` change shouldn't be cherry-picked to the release branch either. The 21.5 version is `8.10`.

Comment (billpittman) [reply conf/distro/nilrt.conf]: Absolutely correct.  I got confused there.  Thanks, and I've added your requested change.

**Review by amstewart — APPROVED** [review]

**Pull request merged**